bitnami/mongodb/values.yaml

```diff
@@ -89,6 +89,18 @@ auth:
   ##
   # existingSecret: name-of-existing-secret
 
+tls:
+ ## Enable or disable MongoDB TLS Support
+  enabled: true
+ ##
+ ## Bitnami Nginix image
+ ##
+  image:
+    registry: docker.io
+    repository: bitnami/nginx
+    tag: 1.19.3
+    pullPolicy: Always
```

Comment:
We should not be setting pullPolicy to always by default, see the rest of images in the values.yaml. And the same for the values production

Reply:
set to IfNotPresent